skimage/viewer/canvastools/painttool.py

```diff
@@ -37,6 +37,24 @@ class PaintTool(CanvasToolBase):
         Overlay of painted labels displayed on top of image.
     label : int
         Current paint color.
+
+    Examples
+    ----------
+    >>> from skimage.data import camera
+    >>> import matplotlib.pyplot as plt
+    >>> from skimage.viewer.canvastools import PaintTool
+    >>> import numpy as np
+
+    >>> img = camera()
+
+    >>> ax = plt.subplot(111)
+    >>> plt.imshow(img, cmap=plt.cm.gray)
```

Comment:
I'm not sure but this line needs to be skipped too from the doctest.
From the travis log
```
File "/home/travis/build/scikit-image/scikit-image/skimage/viewer/canvastools/painttool.py", line 51, in skimage.viewer.canvastools.painttool.PaintTool
Failed example:
    plt.imshow(img, cmap=plt.cm.gray)
Expected nothing
Got:
    <matplotlib.image.AxesImage object at 0x7f70085f0390>
```